Documentation/ldap-connector.md

```diff
@@ -125,6 +125,9 @@ connectors:
 
       # Represents group name.
       nameAttr: name
+
+      #Add also parents of groups
+      inheritance: true
```

Comment:
This doesn't actually define how groups are determined to be a child of another group.

Reply:
Fixed